cdap-docs/build.sh

```diff
@@ -49,7 +49,9 @@ function usage() {
   echo "    clean     Clean up any previous build's target directories"
   echo "    javadocs  Build Javadocs"
   echo "    licenses  Clean build of License Dependency PDFs"
-  echo "    sdk       Build CDAP SDK"
+  echo "    sdk       Build CDAP SDK (includes the Hydrator plugins if Hydrator plugins source is at"
+  echo "                              '${HYDRATOR_PLUGINS_PATH}'"
```

Comment:
in cdap build readme, we don't specify hydrator, we just say additional artifacts dir, you should follow the same.